maintainers/maintainer-list.nix

```diff
@@ -11524,6 +11524,15 @@
     githubId = 131856;
     name = "Arnout Engelen";
   };
+  rafael = {
+    email = "pr9@tuta.io";
+    github = "rafa-dot-el";
+    githubIb = 104688305;
+    keys = [{
+      longkeyid = "ed25519/0x469E3255A40D2AD6";
+      fingerprint = "5F0B 3EAC F1F9 8155 0946 CDF5 469E 3255 A40D 2AD6";
+    }];
+  };
```

Comment:
Can you make this an extra commit, preceeding the one that adds the "package"?

Reply:
Sure, I added it.